code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -419,6 +420,7 @@
 	if(ismob(loc))
 		loc.visible_message("<span class='warning'>The end of [loc.name]\'s hand inflates rapidly, forming a huge shield-like mass!</span>", "<span class='warning'>We inflate our hand into a strong shield.</span>", "<span class='warning'>You hear organic matter ripping and tearing!</span>")
 		playsound(loc, 'sound/effects/bone_break_1.ogg', 100, TRUE)
+		SSblackbox.record_feedback("nested tally", "changeling_powers", 1, list("[name]"))
```

Comment:
Is this not covered by the blackbox record in `/datum/action/changeling/weapon`?

Reply:
Yeah, you're right. That line should get removed.